aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -124,6 +127,7 @@ public static DataStoreConfiguration defaults() throws DataStoreException {
                 .doSyncRetry(DEFAULT_DO_SYNC_RETRY)
                 .observeQueryMaxTime(MAX_TIME_SEC)
                 .observeQueryMaxRecords(MAX_RECORDS)
+            .outboxErrorRestartDelay(DEFAULT_OUTBOX_ERROR_RESTART_DELAY)
```

Comment:
We have a large PR in progress [here](https://github.com/aws-amplify/amplify-android/pull/2201) which will catch all errors generated within the outbox, so we don't want this restart logic.

(Also side note this code adds `outboxErrorRestartDelay` to the class builder. Our process is we don't add items to the public-facing api without adding them to all of the platforms, so we would need buyin from the other platforms to accept that part, just FYI)

Reply:
I'd have to review the other side to understand if I'm comfortable with the efforts to make the outbox processor restart. The code that I concluded with was very effective and did not disrupt the overall orchestrator with stops and starts as the outbox processor is a passive system.

That being said I can remove those components from this PR.